src/OpenTelemetry.Exporter.OneCollector/CHANGELOG.md

```diff
@@ -6,10 +6,14 @@
   syntax.
   ([#1073](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1073))
 
-* Added support for sending common schema `dt` (Distributed Tracing) extensions
+* Added support for sending common schema `dt` (Distributed Tracing) extension
```

Comment:
Shouldn't this be extensions? One for traceId and another for spanId?

Reply:
🤷 The JSON that gets written is `"ext": { "dt": { "traceId": "...", "spanId": "...", "traceFlags": 0 } }` so it is a single `dt` element with two fields. Changed it because singular seemed more accurate, but also don't feel like plural is _that_ wrong.